src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    static final String DATE_FORMAT_PATTERN = "dd.MM.yyyy";
+
+    public boolean inPeriod(String date, String dateFrom, String dateTo) {
```

Comment:
```suggestion
    private boolean isValidDate(String date, String dateFrom, String dateTo) {
```
private methods should be after public ones